### PR TITLE
Add unit tests for `Bundler::Source`

### PR DESCRIPTION
**Pull request by RochesterinNYC**

### CODE REVIEW

File: spec/bundler/source_spec.rb [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      it_behaves_like "the lockfile specs are not relevant"
+    end
+  end
+
+  describe "#can_lock?" do
+    context "when the passed spec's source is equivalent" do
+      let(:spec) { double(:spec, :source => subject) }
+
+      it "should return true" do
+        expect(subject.can_lock? spec).to eq(true)
```
Comment (segiddins) [inline spec/bundler/source_spec.rb]: `expect(subject.can_lock?(spec)).to be_truthy`

File: spec/bundler/source_spec.rb [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      end
+    end
+
+    context "when the passed spec's source is not equivalent" do
+      let(:spec) { double(:spec, :source => double(:other_source)) }
+
+      it "should return false" do
+        expect(subject.can_lock? spec).to eq(false)
```
Comment (segiddins) [inline spec/bundler/source_spec.rb]: `be_falsey`

File: spec/bundler/source_spec.rb [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      end
+    end
+  end
+
+  describe "#include?" do
+    context "when the passed source is equivalent" do
+      let(:source) { subject }
+
+      it "should return true" do
+        expect(subject.include? source).to eq(true)
```
Comment (segiddins) [inline spec/bundler/source_spec.rb]: `expect(subject).to include(source)`

Comment (RochesterinNYC) [inline spec/bundler/source_spec.rb]: Another instance of rspec using a defined method/operation (`include?`) under the hood?

**Comment by RochesterinNYC** [comment]
Updated the PR.

**Comment by indirect** [comment]
👍

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**